inputs.tf

```
@@ -80,4 +80,10 @@ variable "tfstate_acl_bypass" {
   type = list(string)
   description = " (Optional) Specifies whether traffic is bypassed for Logging/Metrics/AzureServices. Valid options are any combination of Logging, Metrics, AzureServices, or None. Defaults to [\"AzureServices\"]."
   default = null
+}
+
+variable "tfstate_entra_group_blob_owner_list" {
+  type = list(string)
+  description = "(Optional) List of Entra ID group object IDs to assign the Storage Blob Data Owner role on the storage account."
+  default = null
```

```suggestion
  default = []
```

Setting default `[]` instead of `null` would allow you to remove one conditional check below.

---

see commit: [84e5289](https://github.com/tamu-edu/it-ae-tfmod-azure-state/pull/21/commits/84e5289b1c7b0ae6fe2b4b21fd8e350dccd2fc2b)